chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -92,25 +92,25 @@ zypper --non-interactive up <%= @neutron_agent %>
 crudini --set /etc/neutron/neutron-openvswitch-agent.conf.d/200-crowbar-upgrade.conf ovs ovsdb_interface vsctl
 crudini --set /etc/neutron/neutron-openvswitch-agent.conf.d/200-crowbar-upgrade.conf ovs of_interface ovs-ofctl
 <% end %>
-systemctl start <%= @neutron_agent %>
+systemctl restart <%= @neutron_agent %>
```

Comment:
> I assume the point @aplanas was raising is that we should not restart the service that is handled by pacemaker

Yes, and that's something we need to worry about as soon as we start working on the compute node HA upgrade.  (It's probably worth to add a checklist item to the associated trello card).
For now I think this PR is ok to be merged.